generator/src/lib.rs

```diff
@@ -2387,7 +2399,7 @@ fn derive_getters_and_setters(
 
         let value = variant_ident("VkStructureType", value);
         quote! {
-            unsafe impl #lifetime TaggedStructure for #name #lifetime {
+            unsafe impl TaggedStructure for #name #opaque_lifetime {
```

Comment:
This is the only lifetime that we could reasonably and unconditionally elide, but I've opted not to.